source/examples/fibcmd/CMakeLists.txt

```diff
@@ -59,6 +59,9 @@ target_include_directories(${target}
     PRIVATE
     ${DEFAULT_INCLUDE_DIRECTORIES}
     ${PROJECT_BINARY_DIR}/source/include
+    
+    $<TARGET_PROPERTY:${META_PROJECT_NAME}::baselib,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
You're right, my system behaves as expected. I think my observation came from my former implementation where I relied on ```get_target_property``` instead of generator expressions.